api/RTCPeerConnectionIceErrorEvent.json

```diff
@@ -29,10 +29,10 @@
             "version_added": null
           },
           "safari": {
-            "version_added": null
+            "version_added": false
```

Comment:
Yep, no sign of `RTCPeerConnectionIceErrorEvent` in WebKit source, and I did some digging and failed to find it under another name either.